pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -472,6 +472,16 @@ type CommonPrometheusFields struct {
 	//
 	// +optional
 	EnforcedLabelValueLengthLimit *uint64 `json:"enforcedLabelValueLengthLimit,omitempty"`
+	// When defined, enforcedKeepDroppedTargets specifies a global limit on the number of targets
+	// dropped by relabeling that will be kept in memory. The value overrides
+	// any `spec.keepDroppedTargets` set by
+	// ServiceMonitor, PodMonitor, Probe objects unless `spec.keepDroppedTargets` is
```

Comment:
```suggestion
	// ServiceMonitor, PodMonitor, Probe and ScrapeConfig objects unless `spec.keepDroppedTargets` is
```

Reply:
Oh, I see that you've copied from the fields above... We can keep it consistent or update the other comments to include ScrapeConfig as well, up to you :)

Reply:
Ya. I thought to create separate PR to update comments for all. Ok to add in this one too if needed. Just kept same way like other limits in this PR